ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -521,23 +529,17 @@ else
                 <div><b>Have feedback?</b> <a asp-Page="/Puzzles/SubmitFeedback" asp-route-puzzleid="@Model.Puzzle.ID" target="_blank">Click this link</a> to open the feedback form.</div>
             }
 
-            @if (embedType > 2) // HTML files or images
+            @if (embedType == 3) // HTML files
             {
-                string urlSuffix = "";
-                if (puzzleFilePath.Contains('?')) 
-                {
-                    urlSuffix += "&";
-                }
-                else
-                {
-                    urlSuffix += "?";
-                }
-                urlSuffix += "print=true";
-                <div><b>Want to print?</b> <a href="@puzzleFilePath@urlSuffix" target="_blank">Click this link</a> for a printable version. Print in color!</div>
+                <div><b>Want to print?</b> <span class="print-link">Click this link</span> for a printable version. Please allow popups!</div>
+            }
+            else if (embedType == 4) // images
+            {
+                <div><b>Want to print?</b> <a href="@puzzleFilePath" target="_blank">Click this link</a> for a printable version. Please allow popups!</div>
             }
             else if (embedType == 1) // PDFs
             {
-                <div><b>Want to print?</b> <a href="@puzzleFilePath" download>Click this link</a> to download the file. Print in color!</div>
+                <div><b>Want to print?</b> <a href="@puzzleFilePath" target="_blank" download>Click this link</a> to download the file. Please allow popups!</div>
```

Comment:
It would be nice to keep the instruction to print in color

Reply:
Agreed, but given how difficult it is to know how to do that, as well as not having a ton of space horizontally, this seemed more important.